articles/cognitive-services/Emotion/QuickStarts/Python.md

```diff
@@ -18,107 +17,143 @@ ms.author: anroth
 > Video API Preview will end on October 30th, 2017. Try the new [Video Indexer API Preview](https://azure.microsoft.com/services/cognitive-services/video-indexer/) to easily extract insights from 
 videos and to enhance content discovery experiences, such as search results, by detecting spoken words, faces, characters, and emotions. [Learn more](https://docs.microsoft.com/azure/cognitive-services/video-indexer/video-indexer-overview).
 
-This article provides information and code samples to help you quickly get started using the [Emotion API Recognize method](https://dev.projectoxford.ai/docs/services/5639d931ca73072154c1ce89/operations/563b31ea778daf121cc3a5fa) with Python to recognize the emotions expressed by one or more people in an image. 
+This walkthrough provides information and code samples to help you quickly get started using the [Emotion API Recognize method](https://westus.dev.cognitive.microsoft.com/docs/services/5639d931ca73072154c1ce89/operations/563b31ea778daf121cc3a5fa) with Python to recognize the emotions expressed by one or more people in an image. 
+
+You can run this example as a Jupyter notebook on [MyBinder](https://mybinder.org) by clicking on the launch Binder badge: 
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/Microsoft/cognitive-services-notebooks/master?filepath=EmotionAPI.ipynb)
+
 
 ## Prerequisite
-* Get your free Subscription Key [here](https://azure.microsoft.com/en-us/try/cognitive-services/)
+Get your free Subscription Key [here](https://azure.microsoft.com/en-us/try/cognitive-services/)
+
+## Running the walkthrough
+Please replace `subscription_key` below with the API key you obtained earlier.
+
+
+```python
+subscription_key = "11966e80bb4a407bac5a340693a790e4"
+assert subscription_key
+```
+
+Also, please verify that the URL below corresponds to the region you used when setting up the API key. If you are using a trial key, you are good to go!
+
+
+```python
+emotion_recognition_url = "https://westus.api.cognitive.microsoft.com/emotion/v1.0/recognize"
+```
+
+In the emotion API walktrough, we will be using images that are stored on disk. You can also use images that are available via a publically-accessible URL. For more details, please see the [REST API](https://westus.dev.cognitive.microsoft.com/docs/services/5639d931ca73072154c1ce89/operations/563b31ea778daf121cc3a5fa) documentation.
+
+Since the image data is passed as part of the request body, notice that we need to set the `Content-Type` header to `application/octet-stream`. If you are passing in an image via a URL, remember to set the header to:
+```python
+header = {'Ocp-Apim-Subscription-Key': subscription_key }
```

Comment:
if using URL, shouldn't the 'Content-Type' header be set to 'application/json'?
and the 'Ocp-Apim-Subscription-Key' should be required in both URL and image data case.